opae.spec.rhel

```diff
@@ -72,10 +71,10 @@ OPAE headers, tools, sample source, and documentation
 %setup -q -n %{name}-%{version}-%{opae_release}
 
 %build
-%cmake -DOPAE_MINIMAL_BUILD=ON 
+%cmake -DOPAE_MINIMAL_BUILD=ON
 
 %if 0%{?rhel}
-  %make_build
+  %cmake_build
```

Comment:
This was per Tom.  Why are we changing it?

Reply:
changes from  %make_build  to %cmake_build because of genearte rpm fails on RHEL 9.0 version